src/pages-and-resources/live/Settings.test.jsx

```diff
@@ -0,0 +1,193 @@
+          <Switch>
+            <PageRoute path={liveSettingsUrl}>
+              <LiveSettings onClose={() => {}} />
+            </PageRoute>
+          </Switch>
+        </PagesAndResourcesProvider>
+      </AppProvider>
+    </IntlProvider>,
+  );
+  return wrapper;
+};
+
+describe('LiveSettings', () => {
+  beforeEach(async () => {
+    initializeMockApp({
+      authenticatedUser: {
+        userId: 3,
+        username: 'abc123',
+        administrator: false,
+        roles: [],
+      },
+    });
+    store = initializeStore(getDefaultStoreState());
+    axiosMock = new MockAdapter(getAuthenticatedHttpClient());
+    history.push(liveSettingsUrl);
+  });
+
+  test('"Configure Live" modal is opening', async () => {
+    const wrapper = await renderComponent();
+    expect(await wrapper.findByRole('dialog')).toBeVisible();
+  });
+
+  test('"Configure Live" heading is displayed', async () => {
+    const wrapper = await renderComponent();
+    const headingElement = (await wrapper.findByRole('dialog')).firstChild;
+    expect(headingElement).toHaveTextContent('Configure Live');
+  });
+
+  test('Title, badge and helper text is displayed when live is enabled', async () => {
+    const wrapper = await renderComponent();
+    const label = wrapper.container.querySelector('label[for="enable-live-toggle"]');
+    expect(label).toHaveTextContent('Live');
+    expect(label.firstChild).toHaveTextContent('Enabled');
+  });
+
+  test('Title and helper text is displayed but not badge when live is disabled', async () => {
+    const newUrl = `${providerConfigurationApiUrl}/${courseId}/`;
+    axiosMock.onGet(newUrl).reply(
+      200,
+      generateLiveConfigurationApiResponse(false, false),
+    );
+    await executeThunk(fetchLiveConfiguration(courseId), store.dispatch);
+    const wrapper = await renderComponent();
+    const label = wrapper.container.querySelector('label[for="enable-live-toggle"]');
+    expect(label).toHaveTextContent('Live');
+    expect(label.firstChild).not.toHaveTextContent('Enabled');
+  });
+
+  test('Provider heading, helper text and providers are displayed', async () => {
+    const url = `${providerConfigurationApiUrl}/${courseId}/`;
+    axiosMock.onGet(url).reply(
+      200,
+      generateLiveConfigurationApiResponse(false, false),
+    );
+    await executeThunk(fetchLiveConfiguration(courseId), store.dispatch);
+    const wrapper = await renderComponent();
+    const providers = wrapper.container.querySelector('div[role="group"]');
+    expect(providers.childElementCount).toBe(1);
+    expect(providers.firstChild.lastChild.lastChild).toHaveTextContent('Zoom');
+    expect(providers.parentNode.querySelector('p')).toHaveTextContent(
+      messages.providerHelperText.defaultMessage.replace('{providerName}', 'Zoom'),
+    );
+  });
+
+  test('When piiSharing is enabled. Helper text, lti fields should be visible. Connect to support should not be visible', async () => {
```

Comment:
Nit: only helper text and LTI field should be visible when PII sharing is enabled.